Jenkinsfile

```diff
@@ -92,6 +97,30 @@ String getLabel() {
   return result
 }
 
+boolean getAllFuncTests() {
+  def paramVal = null
+  try {
+    paramVal = params.allFuncTests
+  } catch (e1) {
+    // workaround for https://issues.jenkins-ci.org/browse/JENKINS-38813
+    echo '[WARNING] unable to access `params`'
+    echo getStackTrace(e1)
+    try {
+      paramVal = allFuncTests
+    } catch (e2) {
+      echo '[WARNING] unable to access `allFuncTests`'
+      echo getStackTrace(e2)
```

Comment:
It only logs two lines at most (and only to the build log) unless there is an exception. 

The exceptions shouldn't happen unless there is a bug (eg https://issues.jenkins-ci.org/browse/JENKINS-38813), but I want to be sure they can't break the build; I also don't want to lose the diagnostic info.